src/platform.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2024 Dylan Baker
+// SPDX-License-Identifier: MIT
+
+#include "platform.hpp"
+
+namespace fs = std::filesystem;
+
+namespace platform {
+
+    const fs::path libdir() {
+        // TODO: libdir needs to be configurable based on the personality,
```

Comment:
Should this be an issue and/or a more noisy warning on stdout?

Reply:
I think that’s: https://github.com/cps-org/cps-config/issues/15, which I hoped we could iterate in cps-config then suggest as a supplementary schema

Reply:
Well, we can always clean this up later, but it'd be nice for TODOs to come with issue identifiers or *something* at some point. They're easy to lose track of.

One option is to use the `#warning` pragma so they're really annoying to maintainers (us).